decorators/placeholder.js

```diff
@@ -6,18 +6,58 @@ var _ = require('lodash'),
   addComponentHandler = require('../services/components/add-component-handler');
 
 /**
+ *
+ * @param {Array} groupItems
+ * @param {string} propName
+ * @returns {object}
```

Comment:
technically yes, but in practical terms we care that it's different. see `_.isObject(null)`